campaign_themes/arctic.json

```diff
@@ -186,5 +189,17 @@
 				}
 			}
 		}
-	]
+	],
+	"hidden_fields": {
```

Comment:
I'm hardcoding these to the JSON files just to make the switch. Not sure if we need more than that for now.

Reply:
Maybe it would make sense to not have this as a separate property, but instead combine them with the configurable fields, making the distinction on the fields themselves using an `editable` attribute or something. That will make it easier and more clear in version history when a field changes from being hidden to being configurable.

Reply:
Maybe it would make sense to separate the fields that contain the value of a css variable, and those that contain values that are used in some backend logic (such as logo and navigation type).

Reply:
Then we would have a property which contains only css variables that have the variable name as a key.